src/Illuminate/Routing/Redirector.php

```diff
@@ -126,6 +126,16 @@ public function setIntendedUrl($url)
         return $this;
     }
 
+    /**
+     * Get the intended url.
+     *
+     * @return string|null
+     */
+    public function getIntendedUrl()
```

Comment:
Would be better to add type hinting

Reply:
Yes, I would love that, but for now I think it's better be consistent. If most of the methods don't have typehints, I'd stick with that.

Reply:
Right now, we don't add type hints like these in the framework. We aim for consistency.